src/transaction_builder.cpp

```diff
@@ -15,6 +15,81 @@
 #include <librustzcash.h>
 #include <rust/ed25519.h>
 
+uint256 ProduceZip244SignatureHash(
+    const CTransaction& tx,
+    const orchard::UnauthorizedBundle& orchardBundle)
+{
+    uint256 dataToBeSigned;
+    PrecomputedTransactionData local(tx);
+    if (!zcash_builder_zip244_shielded_signature_digest(
+        local.preTx.release(),
+        orchardBundle.inner.get(),
+        dataToBeSigned.begin()))
+    {
+        throw std::logic_error("ZIP 225 signature hash failed");
```

Comment:
```suggestion
        throw std::logic_error("ZIP 244 signature hash failed");
```
(I know ZIP 225 is also relevant, but it doesn't actually define the hash.)